linetools/isgm/utils.py

```diff
@@ -376,6 +445,34 @@ def synthesize_components(components, zcomp=None, vbuff=0*u.km/u.s):
     return synth_comp
 
 
+def get_components_at_z(complist, z, dvlims):
+    """In a given list of components, it finds
+    the ones that are within dvlims from a given redshift
+    and returns a list of those components
+
+    Parameters
+    ----------
+    complist : list
+        List of AbsComponents
+    z : float
+        Redshift there to look for components
+    dvlims : Quantity array
+        Rest-frame velocity limits around z
+        to look for components
+
+    Returns
+    -------
+    components_at_z : list
+        List of components within complist within dvlims from z
+    """
+    good_complist = []
```

Comment:
Might do a little error checking here, especially on dvlims
to insure it is a Quantity array with velocity units.